plugins/Jobs.cpp

```diff
@@ -1195,6 +1196,34 @@ bool BedrockPlugin_Jobs::processCommand(SQLite& db, BedrockCommand& command) {
         return true;
     }
 
+    // Requeue a job for which a getJob(s) command could not complete.
+    else if (SIEquals(requestVerb, "RequeueJobs")) {
+        SINFO("Requeueing jobs with IDs: " << command.request["jobIDs"]);
+        list<int64_t> jobIDs = SParseIntegerList(command.request["jobIDs"]);
+
+        SQResult result;
+        string selectQuery = "SELECT jobID, retryAfter FROM jobs WHERE jobID IN(" + SQList(jobIDs)+ ");";
+        db.read(selectQuery, result);
+
+        for (auto& row : result.rows) {
+            int64_t jobID = stoll(row[0]);
+            bool retryAfter = !row[1].empty();
+            string updateQuery;
+            if (retryAfter) {
```

Comment:
Why does it matter if the the job has a `retryAfter` or not? 
I think all the jobs should be updated to `state='QUEUED'` and `nextRun=DATETIME('now())` since they were about to be run anyway

The tricky cases are
1.  `retryAfter` is not empty, but this is actually irrelevant because the job would of been retried again anyway so we might it well retry it now
1.  `repeat` is not empty.  This is a bit trickier.  At first I was thinking we should respect the `repeat` value and therefore was considering proposing that we use `RetryJob` instead of `RequeueJobs` since `RetryJob` handles all the weird cases already. But `repeat` values can be set up so the job runs once a year and it seems silly to wait a year to run a job because the server timed out.

Reply:
Ok, I think just always making them `QUEUED` with a `nextRun` time of right now is simpler and has the same effect.

As far as `repeat` goes, I think I agree with your conclusion - these jobs weren't run actually run and thus this shouldn't affect when they repeat, we should try to run them again as soon as possible, and then they will repeat as normal.